### ISSUE TITLE
No way to import updated version of third party bin.

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#2407]
_From [bitsybof...@gmail.com](https://code.google.com/u/112850805511005419452/) on January 30, 2013 02:22:41_

You have previously imported a bin which contains a large number of parts, eg Adafruit's. https://github.com/adafruit/Fritzing-Library/blob/master/AdaFruit.fzbz Closing Fritzing, you keep the imported parts and save the bin when prompted.

At some time later you wish to update the bin, how do you do this?

The only way is to import the updated bin again this gets you a duplicate bin of the same name.

If you save that bin, as the *.fzb name as you saved the first bin, close and reopen Fritzing, you now have two bins sharing the same fzb.

If you then delete one of the bins, close fritzing and reopen, you have lost both bins because deleting one of them deleted the fzb.

In short, there appears to be no way to cleanly update a third party bin in your fritzing install.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=2408_
